src/main/kotlin/app/revanced/patches/all/misc/mitmproxy/patch/MitmProxyPatch.kt

```diff
@@ -0,0 +1,86 @@
+package app.revanced.patches.all.misc.mitmproxy.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.*
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.all.misc.debugging.patch.EnableAndroidDebuggingPatch
+import org.w3c.dom.Element
+import java.io.File
+
+@Patch(false)
+@Name("Mitm proxy")
```

Comment:
Call this patch "Override certificate pinning". The reason for this is that this patch is not a mitm proxy in itself.